doom-modeline.el

```diff
@@ -227,7 +228,8 @@ If DEFAULT is non-nil, set the default mode-line for all buffers."
 
         ;; Special handles
         (advice-add #'helm-display-mode-line :after #'doom-modeline-set-helm-modeline)
-        (setq helm-ag-show-status-function #'doom-modeline-set-helm-modeline))
+        (setq helm-ag-show-status-function nil
+              helm-fd-mode-line-function nil))
```

Comment:
Should recover `helm-fd-mode-line-function` while disabling `doom-modeline-mode`.